src/libcollections/fmt.rs

```diff
@@ -261,7 +261,9 @@
 //! This and `writeln` are two macros which are used to emit the format string
 //! to a specified stream. This is used to prevent intermediate allocations of
 //! format strings and instead directly write the output. Under the hood, this
-//! function is actually invoking the `write` function defined in this module.
+//! function is actually invoking the
+//! [`write_fmt`](../../io/trait.Write.html#method.write_fmt) function defined on the
```

Comment:
The better thing to test is to build docs and then check by hand:

```
make docs
```

Then open `doc/index.html` with any web browser, go to the modified url and click on it. Then just compare with "real" url and you'll figure out easily.